### ISSUE TITLE
Inertia when submit form

### ISSUE THREAD

**Issue by majweb** [inertiajs/inertia#701]
### Versions:

- `@inertiajs/inertia` version: #.#.#
- `@inertiajs/inertia-react` version: #.#.#

### Describe the problem:
I use laravel with inertia+reactjs.
I use form helpers when I send form with redirect in console i have error with unmounted component. How remove this errors?

**Comment by riquemuramoto** [comment]
Having the same issue. Even the ping-crm demo has the same message when login.

Have you solve this issue?

I'm still learning React JS. How much impact this can have.

**Comment by reinink** [comment]
This has been fixed: #680

Please update to [inertia-react@0.6.0](https://inertiajs.com/releases/inertia-react-0.6.0-2021-05-25). Note, this is a breaking change that requires also updating to [inertia@0.9.0](https://inertiajs.com/releases/inertia-0.9.0-2021-05-25).

**Comment by riquemuramoto** [comment]
Hi

I've updated the packages and still having the issue.

I checked now and only happens when i use the useForm helper.

I'm doing the examples available on the documentation: https://inertiajs.com/forms

When I use the useForm helper it throws the warning of "... unmounted component". Checking the console it's pointing to the useForm hook.

If I make the form with useState and values, setValues, works with no errors. But i would like to use the useful helpers of setData, processing, etc.
